Microsoft.Azure.CosmosRepository/src/IRepository.cs

```diff
@@ -137,6 +138,21 @@ public interface IRepository<TItem> where TItem : IItem
             IEnumerable<TItem> values,
             CancellationToken cancellationToken = default);
 
+
+        /// <summary>
+        ///
+        /// </summary>
+        /// <param name="id"></param>
+        /// <param name="partitionKeyValue"></param>
+        /// <param name="builder"></param>
+        /// <param name="cancellationToken"></param>
+        /// <returns></returns>
+        ValueTask UpdateAsync(
+            string id,
+            string partitionKeyValue,

```

Comment:
```suggestion
            string partitionKeyValue = null,
```

I'd imagine that for parity with the other APIs we'd want this to be `null`, and assigned auto-magically like we do elsewhere?

Reply:
Yes will update this.